recipes/mender-shell/debian-master/control

```diff
@@ -0,0 +1,12 @@
+Source: mender-shell
+Maintainer: Mender Team <mender@northern.tech>
+Section: misc
+Priority: optional
+Standards-Version: 3.9.2
+Build-Depends: debhelper (>= 9)
+
+Package: mender-shell
+Architecture: any
+Depends: libc6 (>= 2.12), libglib2.0-0 (>=2.50), mender-client (>=2.5.0)
+Description: Mender Shell
+ Mender is an open source over-the-air (OTA) software updater for embedded Linux devices.
```

Comment:
suggested: 'Mender Shell is a Mender add-on allowing to open remote interactive shell terminal sessions'

Reply:
@merlin-northern Thanks, I'll change.